swift/ql/lib/swift/File.qll

```diff
@@ -0,0 +1,24 @@
+/** TODO: to be replaced */
```

Comment:
This will be replaced by a combination of generated and hand-written code. Currently, I picked it up to close the "ql test" circle.